api/src/pcapi/routes/pro/venues.py

```diff
@@ -126,13 +118,25 @@ def edit_venue(venue_id: str, body: EditVenueBodyModel) -> GetVenueResponseModel
     if body.bookingEmail and body.isEmailAppliedOnAllOffers:
         update_all_venue_offers_email_job.delay(venue, body.bookingEmail)
 
-    return GetVenueResponseModel.from_orm(venue)
+    return venues_serialize.GetVenueResponseModel.from_orm(venue)
+
+
+@private_api.route("/venues/<venue_id>/pricing_point", methods=["POST"])
+@login_required
+@spectree_serialize(on_success_status=204, api=blueprint.pro_private_schema)
+def post_link_venue_to_pricing_point(venue_id: str, body: venues_serialize.PostPricingPointBodyModel) -> None:
```

Comment:
(détail / question de goût) Préfixer par "post_" ne me semble pas apporter grand chose, et rend le nom de la fonction (un peu) plus dur à lire.

```suggestion
def link_venue_to_pricing_point(venue_id: str, body: venues_serialize.PostPricingPointBodyModel) -> None:
```